doc/content/modules/developer-guide/pages/project_commit_branch_tag_recipe.adoc

```diff
@@ -0,0 +1,158 @@
+= Project and commit recipe
+
+Learn how to manage projects and retrieve related commits programmatically.
+Each recipe includes a detailed explanation, step-by-step instructions, and code snippets.
+
+Recipes covered:
+
+* xref:project_commit_branch_tag_recipe.adoc#_get_projects[Get Projects]: Fetch and display all projects currently available on the server.
+* xref:project_commit_branch_tag_recipe.adoc#_create_a_new_project[Create a New Project]: Create a new project with a unique name.
+* xref:project_commit_branch_tag_recipe.adoc#_get_commits[Get Commits]: Retrieve a list of commits for a specific project.
+
+== Get projects
+This code example demonstrates how to use Python's requests library to interact with the {product} API and retrieve a list of projects.
+It sends a `GET` request to the API endpoint for projects, processes the response, and prints the name and ID of each project.
+
+Example script to fetch projects:
+
+[source,python]
```

Comment:
When I execute this script, I get an error: `ModuleNotFoundError: No module named 'requests'`.
Yet, I installed the required module as expected at prerequisite of this cookbook.

Reply:
It means the installation didn’t work as expected. We need to identify what’s wrong with the instructions in the prerequisites.